### PR TITLE
change the tag of readme of databox

### PR DESCRIPTION
**Pull request by dw511214992**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by zikalino — APPROVED** [review]

File: specification/databox/resource-manager/readme.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ These are the global settings for the DataBox API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2018-01
+tag: package-2019-09
```
Comment (zikalino) [inline specification/databox/resource-manager/readme.md]: could you put package-2019-09 at the top?

File: specification/databox/resource-manager/readme.md [hunk 2]
```diff
@@ -26,25 +26,25 @@ These are the global settings for the DataBox API.
...
 ```
 
-### Tag: package-2018-01
+### Tag: package-2019-09
```
Comment (jsntcy) [inline specification/databox/resource-manager/readme.md]: I think you should add package-2019-09 at the top instead of replacing package-2018-01.

Comment (dw511214992) [reply specification/databox/resource-manager/readme.md]: Hi @jsntcy It's in basic information block and I think we should replace it.

Comment (jsntcy) [reply specification/databox/resource-manager/readme.md]: Sorry, I mislooked it.

---
In reply to: [377524675](https://github.com/Azure/azure-rest-api-specs/pull/8355#discussion_r377524675) [](ancestors = 377524675)

**Review by jsntcy — APPROVED** [review]
:shipit:

**Comment by sashanm** [comment]
Changes looks good

**Comment by dw511214992** [comment]
Hi @erich-wang  Could you please merge this pr? I don't have permission to do it. Thank you.

**Pull request merged**